### ISSUE TITLE
CMD is disabled, hence .bat files are unable to run

### ISSUE THREAD

**Issue by sudokit** [flutter/flutter#137988]
### Is there an existing issue for this?

- [X] I have searched the [existing issues](https://github.com/flutter/flutter/issues)
- [X] I have read the [guide to filing a bug](https://flutter.dev/docs/resources/bug-reports)

### Steps to reproduce

Follow the steps in [Flutter get started](https://docs.flutter.dev/get-started/install/windows) and try running ```flutter```

### Actual results

It results in:
```
PS> flutter

The command prompt has been disabled by your administrator.

Press any key to continue . . .
```

So not actually any bug, just wondering if maybe it would make sense to migrate to powershell script instead of batch. I totally understand if not.

Wasn't either sure if should be put under feature requests, but since it wouldn't be necessarily a flutter feature, i thought id put it here.

### Logs

Can't obtain

### Flutter Doctor output

Can't obtain

**Comment by dam-ease** [comment]
Thanks for filing this. Not sure I fully understand this. You can run this command from either the command prompt or Powershell on Windows.

> So not actually any bug, just wondering if maybe it would make sense to migrate to Powershell script instead of batch. I totally understand if not.

You can check out https://docs.flutter.dev/get-started/install/windows#system-requirements for more information.

**Comment by sudokit** [comment]
Yeah I just mean that if the command prompt is inaccessible (in my case completely blocked by the organization admin), it isn't possible to run .bat files. And batch files are a little old anyways :p
So was just curious if it would make sense to try to migrate the flutter.bat and dart.bat from the bin to .ps1 files(powershell scripts).

**Comment by dam-ease** [comment]
Thanks for your response. Once our end, once CMD is disabled, we can't interact with it anymore, but can still call flutter in PS.
| CMD | PowerShell |
|--------|--------|
| <img src="https://github.com/flutter/flutter/assets/53122008/64b7f328-c922-4cb4-8baf-54bcfcd4d3a5"> | <img src="https://github.com/flutter/flutter/assets/53122008/4f221a87-801e-48b9-b959-b56ddde8330c"> | 

But from your logs in the original comment, I see you are on PS and getting the same message as on CMD.
More info could be gotten from https://github.com/flutter/flutter/issues/7654 as well.
I'll be labelling as a request for more insights from the team.
Thank you!

**Comment by bkonyi** [comment]
@sudokit is this still an issue you're encountering?
